client/wfm/src/process-instance/ProcessInstanceRepository.ts

```diff
@@ -0,0 +1,8 @@
+import {ProcessInstance} from './ProcessInstance';
+export interface ProcessInstanceRepository {
+  getAll(): Promise<ProcessInstance[]>;
```

Comment:
getAllForUser and getAll should use paggination. 
We should add todo to indicate that this needs to be paginated.